presto-docs/src/main/sphinx/release/release-0.165.rst

```diff
@@ -0,0 +1,22 @@
+=============
+Release 0.165
+=============
+
+General Changes
+---------------
+
+* Make ``AT`` non-reserved keyword.
+* Improve performance of :func:`transform`.
+* Improve exchange performance by deserializing in parallel.
+* Add support for compressed exchanges.
```

Comment:
good point, based on the code, it looks like it can be set with exchange_compression